n2y/blocks.py

```diff
@@ -67,6 +67,10 @@ def children_to_pandoc(self):
         for block_type, blocks in groupby(self.children, lambda c: type(c)):
             if issubclass(block_type, ListItemBlock):
                 pandoc_ast.append(block_type.list_to_pandoc(blocks))
+            elif block_type == ChildPageBlock:
```

Comment:
It's not the same because ChildPageBlock.to_pandoc() results in a Pandoc block which will cause an error because all of the pandoc will eventually end up in another Pandoc block. that's why `result[1]` is selected on line 73. I could bring this logic down into that iteration though